themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -1404,29 +1387,23 @@ module.factory('TimeUnit2', function() {
     var t = {};
 
     t.asUnit = function(time) {
-
         var unit = 'Minutes';
-
         if (time) {
-            if (time == -1) {
```

Comment:
Not 100% sure, but looks like those changes in the time is also something, which should be removed from your PR (likely changed by accident when rebasing)